jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
```

Comment:
I think thats fine, the reason I changed it was because it doesn't allow you to do anything without a comment, but thats my case and I could just update my own templates locally.

When I was looking around in there, it seems that our version of jira validates the comment is not just whitespace, and gives us back an error.